src/aircraft_cmd.cpp

```diff
@@ -1693,6 +1693,16 @@ static void AircraftEventHandler_Flying(Aircraft *v, const AirportFTAClass *apc)
 			current = current->next.get();
 		}
 	}
+
+	/**
+	 * If the aircraft is in the air and there are no valid airport left then crash.
+	 */
```

Comment:
Inline comment within a function doesn't need doxygen style.

```suggestion
	/* If the aircraft is in the air and there are no valid airport left then crash. */
```